packages/@tinacms/cli/src/index.ts

```diff
@@ -1,94 +1,33 @@
-import { logText } from './utils/theme'
-export { defineSchema } from './cmds/compile'
-import { logger } from './logger'
+import { DevCommand } from './next/commands/dev-command'
+import { BuildCommand } from './next/commands/build-command'
+import { AuditCommand } from './next/commands/audit-command'
+import { InitCommand } from './next/commands/init-command'
+// import { CodemodCommand } from './next/commands/codemod-command'
+
 export type {
   Schema,
   TinaSchema,
   Collection,
   TinaTemplate,
 } from '@tinacms/graphql'
 
-const program = new commander.Command(name)
-const registerCommands = (commands: Command[], noHelp: boolean = false) => {
-  commands.forEach((command, i) => {
-    let newCmd = program
-      .command(command.command, { noHelp })
-      .description(command.description)
-      .action((...args) => {
-        command.action(...args)
-      })
-
-    if (command.alias) {
-      newCmd = newCmd.alias(command.alias)
-    }
-
-    newCmd.on('--help', function () {
-      if (command.examples) {
-        logger.info(`\nExamples:\n  ${command.examples}`)
-      }
-      if (command.subCommands) {
-        logger.info('\nCommands:')
-        const optionTag = ' [options]'
-        command.subCommands.forEach((subcommand, i) => {
-          const commandStr = `${subcommand.command}${
-            (subcommand.options || []).length ? optionTag : ''
-          }`
-
-          const padLength =
-            Math.max(...command.subCommands.map((sub) => sub.command.length)) +
-            optionTag.length
-          logger.info(
-            `${commandStr.padEnd(padLength)} ${subcommand.description}`
-          )
-        })
-      }
-      logger.info('')
-    })
-    ;(command.options || []).forEach((option) => {
-      newCmd.option(option.name, option.description, option?.defaultValue)
-    })
-
-    if (command.subCommands) {
-      registerCommands(command.subCommands, true)
-    }
-  })
-}
-
-export async function init(args: any) {
-  program.version(version)
-
-  const commands: Command[] = [...baseCommands]
-
-  registerCommands(commands)
-
-  program.usage('command [options]')
-  // error on unknown commands
-  program.on('command:*', function () {
-    logger.error(
-      'Invalid command: %s\nSee --help for a list of available commands.',
-      args.join(' ')
-    )
-    process.exit(1)
-  })
-
-  program.on('--help', function () {
-    logger.info(
-      logText(`
-You can get help on any command with "-h" or "--help".
-e.g: "tinacms server:start --help"
-    `)
-    )
-  })
-
-  if (!process.argv.slice(2).length) {
-    // no subcommands
-    program.help()
-  }
-
-  program.parse(args)
-}
+const cli = new Cli({
+  binaryName: `tinacms`,
+  binaryLabel: `TinaCMS`,
+  binaryVersion: version,
+})
+
+cli.register(DevCommand)
+cli.register(BuildCommand)
+cli.register(AuditCommand)
+cli.register(InitCommand)
+// Uncomment when cloud supports /tina & tina-lock.json
+// cli.register(CodemodCommand)
```

Comment:
@kldavis4 I think we're ready to support the `tina` folder since this PR https://github.com/tinacms/tinacms/pull/3563/files. But I don't recall exactly where we left off. This command is a codemod to move the folder, but if you have a `tina` folder things should work regardless.